src/data/notifications.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  SETTLEMENT: 'New settlement has successfully been created!',
```

Comment:
I think it will better name it SETTLEMENT_SUCCESS, because it is more consistent with meaning of message. 
Same comment for next variables.